hack/istio/install-sleep-demo.sh

```diff
@@ -19,6 +19,8 @@ SCRIPT_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 ISTIO_DIR=
 CLIENT_EXE="oc"
 DELETE_SLEEP="false"
+: ${ISTIO_NAMESPACE:=istio-system}
+: ${ENABLE_INJECTION:=true}
```

Comment:
This seems relevant even for non-Maistra environments. I don't see anything restricting this to maistra-only. Same with the changes in the rest of the script.

Reply:
explicitly not, but implicitly yes, since those ENVs are used only in `prepare_maistra` function which `install-sleep-demo` script uses ,, but maybe it can be used for something else in the future, so, I can backport it into the main branch if needed